Open Government Licence - Ontario.md

```diff
@@ -0,0 +1,68 @@
+# Open Government Licence – Ontario
+
+You are encouraged to use the Information that is available under this licence with only a few conditions.
+
+## Using Information under this licence
+
+1. Use of any Information indicates your acceptance of the terms below.
+2. The Information Provider grants you a worldwide, royalty-free, perpetual, non-exclusive licence to use the Information, including for commercial purposes, subject to the terms below.
+
+### You are free to:
+
+3. Copy, modify, publish, translate, adapt, distribute or otherwise use the Information in any medium, mode or format for any lawful purpose.
+
+### You must, where you do any of the above:
+
+4. Acknowledge the source of the Information by including any attribution statement specified by the Information Provider(s) and, where possible, provide a link to this licence.
+
+If the Information Provider does not provide a specific attribution statement, or if you are using Information from several Information Providers and multiple attributions are not practical for your product or application, you must use the following attribution statement:
+
+Contains information licensed under the Open Government Licence – Ontario.
+
+5. The terms of this licence are important, and if you fail to comply with any of them, the rights granted to you under this licence, or any similar licence granted by the Information Provider, will end automatically.
+
+## Exemptions
+
+6. This licence does not grant you any right to use:
+  
+  1. Personal Information;
```

Comment:
Can these bullets be indented? They should be a nested list to match the [webpage version](https://www.ontario.ca/page/open-government-licence-ontario#section-1).